modules/fs2/src/test/scala/pureconfig/module/fs2Suite.scala

```diff
@@ -0,0 +1,63 @@
+package pureconfig.module.fs2
```

Comment:
Nitpick: this file should be in `modules/fs2/src/test/scala/pureconfig/module/fs2/fs2Suite.scala` (with a fs2 subfolder) in order to match the package name.

Reply:
I feel that is is more important to mirror the src tree, which is `modules/fs2/src/main/scala/pureconfig/module/fs2.scala` as it is a package object, however it is pretty minor so if you insist on it I'm happy to change :)

Reply:
Sure, it makes all sense to mirror the source tree 👍 But then this package should be `pureconfig.module` just like the `fs2` package object has, isn't it?